src/chips/FanChip.js

```diff
@@ -5,6 +5,18 @@ class FanChip {
   #options = {
     // No default options.
```

Comment:
`tapToNavigate`, `tap_action` and `hold_action` should be added as an option with a default value.

In addition, `#options` should be documented in `src\chips\typedefs.js`